.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,45 @@
+name: Create Relaese Pull Request
+
+on:
+  workflow_dispatch:
+    inputs:
+      nextVersion:
+        description: 'Next Version'
+        required: true
+        default: 'vX.Y.Z'
```

Comment:
I think we should remove default not to make the tag vx.y.z by mistake.

Reply:
I'm happy to remove this default If placeholder is supported, but it's not...
(I'd like to tell we need `v` prefix, which is sometimes confusing)

🤔

Reply:
Changed at https://github.com/mackerelio-labs/check-aws-cloudwatch-logs-insights/pull/4/commits/1a2216bb3e9898058ce765a24e2d60b4e99e1aa9 not to use default.